python/tests/api/logger/test_logger.py

```diff
@@ -7,6 +7,8 @@
 
 import whylogs as why
 from whylogs.core import ColumnProfileView
+from whylogs.core.metrics.metrics import MetricConfig
```

Comment:
Should `MetricConfig` be a the `whylogs.core` import level?

Reply:
I like it next to `Metric` where it's used. It would be fishy to import it from a level above `core/metrics`, but it could be made a peer of `metrics.py`. It would be a small file though.